src/main/java/net/sf/jabref/gui/autocompleter/AutoCompleteListener.java

```diff
@@ -410,7 +411,8 @@ private void resetAutoCompletion() {
     }
 
     private String[] findCompletions(String beginning) {
-        return completer.complete(beginning);
+        List<String> results = completer.complete(beginning);
+        return results.toArray(new String[results.size()]);
```

Comment:
Yes it would make sense. However I see `AutoCompleteListener` as deprecated and it should be replaced by `AutoCompleteSupport` in the future.